Disable v8 system instrumentation on Darwin

On Darwin, the v8 system instrumentation requires the header "os/signpost.h"
which is available since apple_sdk 11+. See: https://github.com/nodejs/node/issues/39584

--- old/tools/v8_gypfiles/features.gypi
+++ new/tools/v8_gypfiles/features.gypi
@@ -62,7 +62,7 @@
       }, {
         'is_component_build': 0,
       }],
-      ['OS == "win" or OS == "mac"', {
+      ['OS == "win"', {
         # Sets -DSYSTEM_INSTRUMENTATION. Enables OS-dependent event tracing
         'v8_enable_system_instrumentation': 1,
       }, {
